lib/stripe/api_operations/request.rb

```diff
@@ -0,0 +1,32 @@
+module Stripe
+  module APIOperations
+    module Request
+      module ClassMethods
+        @@opts_to_persist = Set.new([:api_key, :api_base, :stripe_account, :stripe_version])
+
+        def request(method, url, params={}, opts={})
```

Comment:
I couldn't figure out a way to keep this as a protected method and still have it callable from its own instances :-/. Not sure such a way exists.